packages/ses/package.json

```diff
@@ -55,24 +55,34 @@
         "default": "./dist/ses.cjs"
       }
     },
+    "./hermes": {
+      "require": {
+        "types": "./dist/types.d.cts",
+        "default": "./dist/ses-hermes.cjs"
+      }
+    },
     "./tools.js": "./tools.js",
     "./assert-shim.js": "./assert-shim.js",
     "./lockdown-shim.js": "./lockdown-shim.js",
     "./compartment-shim.js": "./compartment-shim.js",
     "./package.json": "./package.json"
   },
   "scripts": {
-    "build": "node scripts/bundle.js",
+    "build:vanilla": "node scripts/bundle.js",
+    "build:hermes": "SES_BUILD_TYPE=hermes node scripts/bundle.js",
```

Comment:
this will fail in powershell (if it matters)

consider [util.parseArgs()](https://nodejs.org/api/util.html#utilparseargsconfig) instead of env vars

Reply:
finally setup dev'ing on windows again woo (winget github.cli, git.git, open.nodejs, set exec policy RemoteSigned, corepack enable 💥)

Reply:
any idea why it's working for me?

![image](https://github.com/user-attachments/assets/3c9493af-9706-44b7-bc9c-0e1b0bdc0dd1)